website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -97,6 +97,12 @@ The following changes should be present:
 - `website/content/en/docs/upgrading-sdk-version/v1.3.0.md`: commit changes (created by changelog generation).
 - `website/config.toml`: commit changes (modified by release script).
 
+Then, run the test-sanity `make` target to regenerate the testdata samples. Note that the sanity tests will fail but the scaffolding in `testdata/` will be generated. 
```

Comment:
Why not run make generate which is the target that generates the scaffolds instead of the test-sanity?

Reply:
/hold

Reply:
+1 `make generate` Under the hood `test-sanity` is calling generate, but the tests fail if a diff is produced.